### PR TITLE
add test_compositing.py

### PR DESCRIPTION
**Pull request by earney**
move distribution to Trusty..
get a recent version of ffmpeg since trusty has very old version, which moviepy complains about..

### CODE REVIEW

File: tests/test_compositing.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import pytest
+from moviepy.editor import *
+
+def test_clips_array():
+    _red = ColorClip((1024,800), col=(255,0,0))
```
Comment (mbeacom) [inline tests/test_compositing.py]: Internal vars?

Comment (earney) [inline tests/test_compositing.py]: @mbeacom .. sorry, I'm not following you.  What do you mean?

Comment (mbeacom) [inline tests/test_compositing.py]: @earney https://shahriar.svbtle.com/underscores-in-python#single-underscore-before-a-name-eg-code-class_1

Comment (mbeacom) [inline tests/test_compositing.py]: And referencing PEP-8 http://stackoverflow.com/a/6930223/2560922

**Comment by earney** [comment]
@mbeacom.  I always append my local variables with an underscore.  Is that discouraged?
As the article says, I also do that for variables I consider private, and shouldn't be used externally, but that doesn't apply here.

**Comment by mbeacom** [comment]
I was curious about the reasoning behind the naming convention. All good.

**Comment by earney** [comment]
if we decide on a naming convention, I'm fine with trying to follow it.  It may take some time for me to break old habits, but I can try. :)

**Comment by Zulko** [comment]
I have never seen such a systematic use of underscores before, are there projects that use this ? There are some rare cases where it helps understanding which variables are 'permanent' and which are throw-away, but I'm not sure they help here.

**Comment by kerstin** [comment]
I've never seen underscores used in front of variable names in Python, just as separators between several words making up one name.

**Pull request merged**